lib/Data.js

```diff
@@ -0,0 +1,84 @@
+export class Test {
```

Comment:
Do we need a class for this? Since there are no methods, an object literal will do just as well.